test/web-platform-tests/to-upstream/dom/nodes/ParentNode-querySelector-All-content.html

```diff
@@ -0,0 +1,377 @@
+<!DOCTYPE html>
```

Comment:
Are these all from-scratch tests that we should work to upstream?? Or are derived from existing tests?

Reply:
They are copied from /dom/nodes/
Should we renamethem too?

Reply:
Why can't we just use the original dom/nodes ones?

Reply:
Both files are called within `to-upstream/dom/nodes/ParentNode-querySelector-All-dont-upstream.html`, `to-upstream/dom/nodes/ParentNode-querySelector-All-xht-dont-upstream.xht`.

```
<script src="selectors.js"></script>
<script src="ParentNode-querySelector-All.js"></script>
```

Is the following what you are suggesting?:

* Change those two paths to absolute paths like
  ```
  <script src="/dom/nodes/selectors.js"></script>
  <script src="/dom/nodes/ParentNode-querySelector-All.js"></script>
  ```
* In `run-single-wpt.js`, make modifications in the same way as `/resources/testharness.js` so that they are loaded from the `./tests` folder

Reply:
The file I am commenting on here is `ParentNode-querySelector-All-content.html`, not `selectors.js` or `ParentNode-querySelector-All.js`. I am asking whether `ParentNode-querySelector-All-content.html` (and `ParentNode-querySelector-All-content.xht`) are completely new tests, or are derived from existing tests.

---

Separately, if we can reduce duplication for `selectors.js` and `ParentNode-querySelector-All.js` using a technique like what you describe, I think that would be good.

Reply:
Ah, they are loaded in iframe.

ParentNode-querySelector-All-dont-upstream.html
L19:
```
frame.src = "ParentNode-querySelector-All-content.html#target";
```

ParentNode-querySelector-All-xht-dont-upstream.xht
L21:
```
frame.src = "ParentNode-querySelector-All-content.xht#target";
```

I think we can remove them and load from `./tests` too.

Reply:
Loading xht (xhtml) files from `./tests` didn't work well.
It seemed to be a content-type issue, but maybe I'm just doing things wrong.